config/initializers/themes/conferences_primer_theme.rb

```diff
@@ -184,6 +188,11 @@
     title: 'Events',
     description: 'Contains details of past and upcoming events',
     parts: %w[text events_list]
+  }, {
+    name: 'embedded_form',
+    title: 'Embedded Form',
```

Comment:
I think Spina generally uses sentence case for titles.
```suggestion
    title: 'Embedded form',
```